### ISSUE TITLE
setup/install - Eek, something is wrong.

### ISSUE THREAD

**Issue by Tek42** [plainmade/unmark#31]
OK, so I've read previous Issues that you've closed regarding this but I figured I would give it a shot.

I initially installed unmark in a subfolder which I understand is not how you recommend it to be set up. So I set up a subdomain and put the files in the root folder for the subdomain. I am an experienced Apache admin so I'm pretty sure I have it all configured  correctly. When I visit my subdomain website I get the error page (Eek something is wrong), which is to be expected since you need to run the install first. So I tried to go to my subdomain/setup but I again got the (Eek) message.  I checked the .htaccess file included with unmark and it looks fine and my server is reading it.

I do run my websites, including the new subdomain I set up on https (to get around a port 80 restriction) so the only thing I can think is that maybe the unmark software cant run over a SSL connection ?

I looked at the routes.php file but really couldnt figure out how to edit it to to possibly make it work. I am not familiar with the CodeIgniter but I did read some of the documentation for the routes.php and it seemed to me that I wouldnt need to edit it if I was running the site in the root folder.

Anyway, if you or someone else who has it running could point me in the right direction I'm sure I can get it working. Thanks !

Mike

**Comment by Darice** [comment]
New year and reorganizing bookmarks I decided to give unmark another try. Clean install, clean database. Same error as above. After digging around I found the problems and fix. 

First:  I had to chmod application/logs/  to 777.  It was set by default to 755   After that error log started being generated. 

Second: On /setup tables are created but only one with the asigned prefix. Hence the error when going to /install  It can't find the tables missing the assigned prefix.
Solution: remove prefix from database.php and setup again without it.

No idea why only one table is created with the prefix and the rest not. No error was generated on that in the error log.
